build/artifacts/Dockerfile

```diff
@@ -10,17 +10,18 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
-FROM golang:1.14-alpine3.12 as builder
-RUN apk add --no-cache ca-certificates
-RUN adduser -D -g '' appuser
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/go-toolset
+FROM registry.access.redhat.com/ubi8/go-toolset:1.15.7-11 as builder
+USER root
 WORKDIR /build/che-plugin-broker/brokers/artifacts/cmd/
 COPY . /build/che-plugin-broker/
-RUN CGO_ENABLED=0 GOOS=linux go build -mod vendor -a -ldflags '-w -s' -installsuffix cgo -o artifacts-broker main.go
+RUN adduser appuser && \
+    CGO_ENABLED=0 GOOS=linux go build -mod vendor -a -ldflags '-w -s' -installsuffix cgo -o artifacts-broker main.go
 
 
-FROM alpine:3.12
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
+FROM registry.access.redhat.com/ubi8-minimal:8.4-200.1622548483
```

Comment:
Can we use micro image there ?

Reply:
I would vote for minimal if it's small enough.
If we go with micro, I assume we'll lose the ability to exec inside a container and debug some issues, but I may be wrong.
Particularly for plugin broker - I haven't debugged it for a while, so my comment may be not applicable here.

Reply:
mcro has bash

Reply:
and the image alpine had sh so I would say we don't change anything

Reply:
I had must try micro before sharing such a concern )

Reply:
switched to micro. let's see what breaks! :D

Reply:
FYI, using the ubi8-micro doesn't work in brew for two reasons:

a) the internally available container name is ubi8/ubi-micro, so need to use that if we want to find the base image
b) some internal process needs to run /bin/rpm on the built container, which fails as ubi8-micro has no package manager installed.

Reply:
well for scratch images it can't run RPM as well so this shouldn't be an issue for micro

Reply:
I suspect there are different rules for Brew builds when the base image is a scratch, than when they're anything else.